utils/utils.go

```diff
@@ -367,3 +369,56 @@ func GetLocalIP() (string, error) {
 func GetFuncName(fn interface{}) string {
 	return runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name()
 }
+
+func WritePidToFile(pid int, path string) error {
```

Comment:
Ideally the `etcdmdmt` package will also the daemon package to manage the etcd daemon. But we are planning to move to embedding etcd. In any case, this will not provide extra utility being here.